modules/users/server/controllers/users/users.password.server.controller.js

```diff
@@ -28,17 +28,23 @@ exports.forgot = function (req, res, next) {
     },
     // Lookup user by username
     function (token, done) {
-      if (req.body.username) {
+      if (req.body.userhandle) {
+
+        var userhandle = req.body.userhandle.toLowerCase();
+
         User.findOne({
-          username: req.body.username.toLowerCase()
+          $or: [
+            { username: userhandle },
```

Comment:
Should we keep this feature consistent with the Login, and use `usernameOrEmail`?